src/Nethermind/Nethermind.Consensus.Clique/CliqueBlockProducer.cs

```diff
@@ -445,7 +446,8 @@ bool IBlockProducer.IsProducingBlocks(ulong? maxProducingInterval)
             header,
             selectedTxs,
             Array.Empty<BlockHeader>(),
-            spec.WithdrawalsEnabled ? Enumerable.Empty<Withdrawal>() : null
+            spec.WithdrawalsEnabled ? Enumerable.Empty<Withdrawal>() : null,
+            spec.DepositsEnabled || spec.IsEip7002Enabled ? Enumerable.Empty<ConsensusRequest>() : null
```

Comment:
why ``spec.DepositsEnabled || spec.IsEip7002Enabled`` wouldn't make sense if it was ``spec.ConsensuRequestsEnabled`` and in IReleaseSpec we have the property there?

Reply:
yeah, wanted to do it, thx. Changed